### ISSUE TITLE
Translation: R-gapminder meta-data and objectives

### ISSUE THREAD

**Issue by TomKellyGenetics** [swcarpentry-ja/i18n#84]
Translation of metadata for the R lessons
-----

A web-preview is available for the lessons so far:https://swcarpentry-ja.github.io/r-novice-gapminder/ja/index/index.html

It would be great if we can get the meta-data translated as well to demonstrate the system with PO files. Are these in the files somewhere?

- table of contents

- pre-requisites

- home page

- lesson objectives

-----

* Please assign yourself to the issue **before** you start translating, so other people knows that you are translating
* Once you have translated, send a pull request and reference this issue

-----

* あなたが翻訳し始めたと分かるように、翻訳を**始める前**にこのイシューを自分に割り振って(assign)して下さい。
* 翻訳が終わったら、プルリクエストを開いて、このイシューを参照・リンクして下さい。

https://github.com/swcarpentry-ja/i18n/blob/5f1d2d19ff9db757f4ed5729308e0a9425723b0c/po/r-novice-gapminder.ja.po

**Comment by joelnitta** [comment]
- Table of Contents: It looks like the table of contents is auto-generated from the YAML front matter in each episode's md file. For some reason nobody translated those lines when they translated the episodes, so we don't see translations showing up in either the table of contents or the translated episode. I will probably file a separate issue (or issues) for this.

- Prerequisites: These have already been translated, but may need to be reviewed.
https://github.com/swcarpentry-ja/i18n/blob/f4aed5e805895a613b0cdffffefd0431a75ac54d/po/r-novice-gapminder.ja.po#L21183-L21193

- Home page: I filed #105 to translate `index.md`

- Lesson objectives: These are also contained in the YAML front matter of each episode.

**Comment by TomKellyGenetics** [comment]
Worth checking the "dictionary" file as well. The po4gitbook script uses this will fill in bits and pieces in each lesson. I think we already discussed this in #41 but changes may be needed here if you see a problem in the web pages. https://github.com/swcarpentry-ja/i18n/blob/ja/lang.md

**Comment by TomKellyGenetics** [comment]
Once it's ready I can give the scripts a whirl and try building an updated version of the webpages.

**Comment by joelnitta** [comment]
Closing this as it is now covered by issues #105, #111, and #112
